app/main/presenters/service_presenters.py

```diff
@@ -24,14 +24,15 @@ def _add_as_attribute_if_key_exists(self, key, service_data):
         if key[0] in service_data:
             setattr(self, key[1], service_data[key[0]])
 
-    def __init__(self, service_data, service_questions):
+    def __init__(self, service_data, service_questions, lots_by_slug):
         self.service_questions = service_questions
         # required attributes directly mapped to service_data values
         self.title = service_data['serviceName']
         self.serviceSummary = service_data['serviceSummary']\
             if 'serviceSummary' in service_data else \
             service_data['serviceDescription']
-        self.lot = service_data['lot']
+        # according to our test data, lot field in service_data may not be correct slug i.e. it may be mixed case
```

Comment:
What test data? I think all our production data had their lot slugs migrated to all-lowercase a long time ago, so I don't think this should be necessary.  Would be good to know if this is really a thing still.

The commit message here suggests it shouldn't be and that all mixed-case lot slugs can be done away with: https://github.com/alphagov/digitalmarketplace-frameworks/commit/b29692060e3c7f90bfb37f5c166ca92850b58d8d